packages/lune/src/commands/ci.ts

```diff
@@ -1,14 +1,16 @@
 import type { Command } from ".";
+
 import { hrtime } from "process";
 import { resolve } from "path";
 import { existsSync } from "fs";
 import { readdir, rm } from "fs/promises";
-import { buildPlugin, loadCfg } from "../builder.js";
+import { buildPlugin } from "../builder";
+import { loadCfg, loadNearestCfgOrDefault } from "../config";
```

Comment:
```suggestion
import { loadCfg, loadNearestCfgOrDefault } from "../config.js";
```
I know it works when bundled but please leave the .js suffix on, as otherwise, node won't run it direct from sources/modules correctly.

This applies to all other removals of the `.js` in an import.

Reply:
Okidoki, fixed with 2b39ee0